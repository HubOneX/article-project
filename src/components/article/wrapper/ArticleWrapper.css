.article-wrapper {
  margin: 40px 50px;
  display: flex;
  flex-direction: column;
  background-color: hsla(0, 0%, 75%, 0.11);
  border: 1px solid hsla(0, 0%, 75%, 0.25);
  border-radius: 20px;
  box-shadow: 0 0 20px 1px rgba(0, 0, 0, 0.5);
  backdrop-filter: blur(15px);
  overflow: hidden;
  max-width: 940px;

  & .article-image {
    max-width: 940px;
  }

  @media screen and (max-width: 960px) {
    max-width: 940px;

    .article-image {
      max-width: 90vw;
    }
  }
}
