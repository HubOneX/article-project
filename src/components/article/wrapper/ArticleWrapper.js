import "./ArticleWrapper.css";

const ArticleWrapper = ({ children }) => {
  return <div className="article-wrapper">{children}</div>;
};

export default ArticleWrapper;
