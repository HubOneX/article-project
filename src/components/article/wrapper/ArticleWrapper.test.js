import renderer from "react-test-renderer";
import ArticleWrapper from "./ArticleWrapper";

describe("ArticleWrapper", () => {
  it("should match the snapshot", () => {
    const wrapper = renderer.create(
      <ArticleWrapper>
        <h1>Element</h1>
      </ArticleWrapper>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
