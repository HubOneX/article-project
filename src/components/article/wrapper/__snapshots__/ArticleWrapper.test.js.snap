// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleWrapper should match the snapshot 1`] = `
<div
  className="article-wrapper"
>
  <h1>
    Element
  </h1>
</div>
`;
